sdk/workloads/Azure.ResourceManager.Workloads/tests/Tests/SviCRUDTests.cs

```diff
@@ -0,0 +1,268 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.Identity;
+using Azure.Security.KeyVault.Secrets;
+using Azure.ResourceManager.Resources;
+using Azure.ResourceManager.Workloads.Models;
+using System.Security.Cryptography.X509Certificates;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using NUnit.Framework;
+using System.Threading;
+
+namespace Azure.ResourceManager.Workloads.Tests.Tests
+{
+    [TestFixture]
+    public class SviCRUDTests : WorkloadsManagementTestBase
+    {
+        /// <summary>
+        /// The Get Ops Extension Status Interval.
+        /// </summary>
+        private const int GetOpsIntervalinMillis = 30000;
+
+        public SviCRUDTests(bool isAsync) : base(isAsync)
+        { }
+
+        [OneTimeTearDown]
+        public void Cleanup()
+        {
+            CleanupResourceGroups();
+        }
+
+        /// <summary>
+        /// The SVI CRUD test that creates and installs
+        /// an SVI.
+        /// </summary>
+        [TestCase]
+        [RecordedTest]
+        public async Task TestSVICrudOperations()
+        {
+            // Create Test RG
+            var resourceGroupName = Recording.GenerateAssetName("SdkRg-SVICrud");
+            SubscriptionResource subscription = await Client.GetDefaultSubscriptionAsync();
+            ResourceGroupResource rg = await CreateResourceGroup(
+                subscription,
+                resourceGroupName,
+                AzureLocation.EastUS);
+
+            SapVirtualInstanceData SviData = this.GetSingleServerPayloadToPut(rg.Data.Name, false);
+
+            SapVirtualInstanceData SviInstallData = this.GetSingleServerPayloadToPut(rg.Data.Name, true);
+
+            // Create SVI
+            try
+            {
+                ArmOperation<SapVirtualInstanceResource> resource = await rg.GetSapVirtualInstances().CreateOrUpdateAsync(
+                    waitUntil: WaitUntil.Completed,
+                    sapVirtualInstanceName: "F95",
+                    data: SviData);
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine(ex);
+            }
+
+            SapVirtualInstanceCollection collection = rg.GetSapVirtualInstances();
+
+            // Install SVI
+            try
```

Comment:
let's move these to separate methods

Reply:
Separate methods for install and create?

Reply:
yes

Reply:
This was done in the same method because we have to create before install anyway right, create + install is the flow used in our E2Es as well, there are no standalone install tests.